sdk/automation/arm-automation/src/operations/nodeCountInformation.ts

```diff
@@ -30,28 +30,35 @@ export class NodeCountInformation {
    * Retrieve counts for Dsc Nodes.
    * @param resourceGroupName Name of an Azure Resource group.
    * @param automationAccountName The name of the automation account.
+   * @param countType The type of counts to retrieve. Possible values include: 'status',
+   * 'nodeconfiguration'
    * @param [options] The optional parameters
    * @returns Promise<Models.NodeCountInformationGetResponse>
    */
-  get(resourceGroupName: string, automationAccountName: string, options?: msRest.RequestOptionsBase): Promise<Models.NodeCountInformationGetResponse>;
```

Comment:
Looks like the countType that was being passed to the client now gets passed just to the get api that needs it